File: c/800/136A.c
//Aadityaraj Kaushal going God Mode here </>
//https://codeforces.com/problemset/problem/136/A

#include <stdio.h>
#include <stdlib.h>





int main(){
    int n;
    scanf("%d", &n);

    int* who_gave = (int*) malloc (sizeof(int) * n);
    int* who_recieved = (int*) malloc (sizeof(int) * n);

    for(int i=0 ; i<n ; i++){
        scanf("%d" , who_gave+i);
        who_recieved[who_gave[i]-1] = i+1;
    }

    for(int i=0 ; i<n ; i++){
        printf("%d " , who_recieved[i]);
    }
    printf("\n");

    return 0;
}

File: python/800/136A.py
"""
    Author : Aadityaraj Kaushal
    Published : 04.03.2025
    Link : https://codeforces.com/problemset/problem/136/A
"""

n = int(input())
input_str = input().split(" ")

for i in range(n) :
    input_str[i] = [input_str[i], 0]

for i in (range(n)) :
    input_str[int(input_str[i][0])-1][1] = i+1

for i in range(n) :
    print(input_str[i][1], end=" ")

